### ISSUE TITLE
Load _instanceName in nested entity failed with unfetched attr exception

### ISSUE THREAD

**Issue by vadimbasko** [Haulmont/jmix-graphql#30]
We need to define which nested entities has `_instanceName` attribute in query and programatically add fields _instanceName depends on in fetch plan. 

I.e. instance name of Car
```
    @InstanceName
    @DependsOnProperties({"manufacturer", "model"})
    public String getDisplayName() {
        return String.format("%s - %s", manufacturer, model);
    }
```

Try to load garages with cars
```
{
  garageList {
    cars {
      _instanceName
    }
  }
}
```

Result - exception, because there are no `manufacturer` and `model` in Car query. 
Query below will works fine

```
{
  garageList {
    cars {
      _instanceName
      manufacturer
      model
    }
  }
}
```

**Comment by vadimbasko** [comment]
Also condition improvement required in `EntityMutationDataFetcher` entity reload after commit. This condition should take in mind, that `entityStates.isLoadedWithFetchPlan` is not always works correctly, if we manually add fields required for `_instanceName`.

**Comment by vadimbasko** [comment]
fixed as part of #20
